src/data/finite/card.lean

```diff
@@ -62,6 +62,9 @@ finite.card_pos_iff.mpr h
 
 namespace finite
 
+lemma cast_card_eq_mk {α : Type*} [finite α] : ↑(nat.card α) = cardinal.mk α :=
+cardinal.cast_to_nat_of_lt_aleph_0 (cardinal.lt_aleph_0_of_finite α)
```

Comment:
I think this can be split into two lemmas: one is the analogue of [cardinal.mk_to_nat_eq_card](https://leanprover-community.github.io/mathlib_docs/set_theory/cardinal/basic.html#cardinal.mk_to_nat_eq_card) with `nat.card` in RHS and without `fintype` hypothesis; the other is that ↑ cancels `to_nat` assuming lt_aleph_0.

Reply:
Isn't the second lemma you mentioned exactly `cardinal.cast_to_nat_of_lt_aleph_0`?

Reply:
I missed that the first lemma is exactly the definition of `nat.card` 😂 
This lemma looks all good to me!